### PR TITLE
docs: add documentation for ImageReader

### PR DESCRIPTION
**Pull request by theUnkownName**
This PR is for [JINAHUBV1-359](https://jinaai.atlassian.net/browse/JINAHUBV1-361?atlOrigin=eyJpIjoiN2FmYjM2ZTFhOTMxNDcxMGI4YzA2NmVkOWM3OTFkYjMiLCJwIjoiaiJ9).

In the Manifest file:

- [x] short description
- [x] points to the README (URL) of the respective Image
- [x] Correct keywords

In the README:

- [x] 1-2 sentence description of the "Executor" and the functionality of it
- [x] Example Section with code snipped of  "how to use it in flow"

### CODE REVIEW

File: crafters/image/ImageReader/README.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
 # ImageReader
 
-loads the image from the given file path and save the `ndarray` of the image in the Document. 
\ No newline at end of file
+**ImageReader** is a crafter that loads an image from a given file path and save the `ndarray` of the image in the Document. It reads the image specified in `buffer` and save the `ndarray` of the image in the `blob` of the document.
```
Comment (JoanFM) [inline crafters/image/ImageReader/README.md:3]: As far as I remember it also can work from bytes right?

File: crafters/image/ImageReader/README.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+
+The **ImageReader** executor needs only one parameter:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `channel_axis`  |It's the desired audio signal to be normalized. |
```
Comment (JoanFM) [inline crafters/image/ImageReader/README.md:9]: Is not an audi signal, is the color channel axis of the resulting image matrix

File: crafters/image/ImageReader/manifest.yml [hunk 3]
```diff
@@ -2,12 +2,11 @@ manifest_version: 1
 name: ImageReader
 kind: crafter
 type: pod
-description: loads the image from the given file path and save the `ndarray` of the
-  image in the Document.
+description: Loads image from given file path and save the `ndarray` of the image in the Document.
```
Comment (JoanFM) [inline crafters/image/ImageReader/manifest.yml:5]: may load also from bytes

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: crafters/image/ImageReader/README.md (hunk 2 above)
Comment (theUnkownName) [reply crafters/image/ImageReader/README.md]: Yes! it was mixed up with the audio file

File: crafters/image/ImageReader/README.md (hunk 1 above)
Comment (theUnkownName) [reply crafters/image/ImageReader/README.md]: Yeah, I changed it already

**Review by JoanFM — APPROVED** [review]

**Pull request merged**